tools/check_lint_rules_identity/lib/src/utils/version_utils.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:pub_semver/pub_semver.dart';
+
+// Version.parse cannot be used if minor does not exist, so I created my own parse method.
+Version parseStringToVersion(String versionString) {
+  final version = versionString.split('.');
+  try {
+    final major = int.parse(version[0]);
+    final minor = int.parse(version[1]);
+    const patch = 0;
+
+    return Version(
+      major,
+      minor,
+      patch,
+    );
+  } on FormatException catch (e) {
+    throw FormatException('Could not parse "$versionString". ${e.message}');
+  }
+}
```

Comment:
![nits-badge](https://img.shields.io/badge/review-nits-orange.svg) 
I thought there might be a possibility of a `RangeError` occurring at the `version[1]` part, which could make it a bit harder to notice the mistake.

![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg) 
How about implementing it in a similar way to the `Version` class, as follows?
```dart
final _versionRegExp = RegExp(r'^(\d+)\.(\d+)$');

// Version.parse cannot be used if patch does not exist, so I created my own parse method.
Version parseStringToVersion(String versionString) {
  final match = _versionRegExp.firstMatch(versionString);
  if (match == null) {
    throw FormatException('Could not parse "$versionString".');
  }

  try {
    final major = int.parse(match[1]!);
    final minor = int.parse(match[2]!);
    const patch = 0;

    return Version(
      major,
      minor,
      patch,
    );
  } on FormatException catch (e) {
    throw FormatException('Could not parse "$versionString". ${e.message}');
  }
}
```

Reply:
Thanks for the suggestion! I agree with you completely.
I fixed. 393b4c17f44855e10fb503c01cc5da299c795e7c

Since it is clear from the type that the argument is a String, the argument name was changed from versionString to version.
4fbba8068843249f9b26951a7e02cb2763b540cc